clusters/k8s-infra-dev-cluster-turnup/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+project      = "k8s-infra-dev-cluster-turnup"
```

Comment:
Is it fair to say that `k8s-infra` is a prefix, `dev` is the environment and `cluster-turnup` is what gives the unique name to the project? If so, I'd split these into multiple variables and document the naming convention for the project somewhere

Reply:
it's a great suggestion! I think it's better to put this as an issue since we have no real hierarchy or convention and it may take some time to implement this.